### DISCUSSION TITLE
Diagonal Selection Tool And Or Direct Lines

### DISCUSSION THREAD

**Discussion by fauxpinpricks** [Zrips/CMI#6470] [Feature Requests]
The request is simple. allow for more election points to create a specific portal polygon shape. much the same way the polygon selection tool works with another known plugin used to modify large amounts of blocks at the same time.

In short. being able to create / diagonal portals or maybe even V shaped portals. thats the same portal. currently in order to make this
one has to create individual portal segments that takes the traveler to their final destination.

It could also be good for those who use cmi portals for particle effects to be able to create a direct path from 1 block to another instead 
of creating many individual portal segments where there could be just one line going from say x:1 y:1 z:1___ to___ x:10 y:8 z:15

**Comment by Zrips** [comment]
It would be simpler to create completely new system for particles only which would allow to have better optimization options as we no longer need to check for player entering area, which is relatively easy and cheap on cuboid area but gets exponentially more complex and expensive with any other shape
